lib/Suggestions.js

```diff
@@ -29,12 +50,12 @@ class Suggestions extends Component {
     const { props } = this;
     const suggestions = props.suggestions.map(function (item, i) {
       return (
-        <li key={i}
-        onMouseDown={props.handleClick.bind(null, i) }
-        onMouseOver={props.handleHover.bind(null, i) }
-        className={i == props.selectedIndex ? "active" : ""}>
-        <span dangerouslySetInnerHTML={this.markIt(item, props.query) } />
-        </li>
+          <li key={i}
```

Comment:
Would we expect the options of this dropdown to change dynamically? It seems like a reasonable assumption. Can use `item` as the key, or does `item` have any properties that can be used as a key?

https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318#.woflu8at1